tests/update_images/test_aws.py

```diff
@@ -39,6 +41,24 @@ def test_describe_images() -> None:
     )
 
 
+def test_parse_name_of_all_images() -> None:
+    """Test AWS parse image name with real aws data."""
+    images = []
+    with open("tests/update_images/testdata/aws_list_images.json") as json_file:
+        images = json.load(json_file)["Images"]
+
+    for image in images:
+        data = aws.parse_image_name(image["Name"])
+        assert data["version"]
+        assert data["virt"]
+        assert data["date"]
+        assert data["arch"]
+        assert data["virt"]
```

Comment:
Just a duplicate

Reply:
oh thank you for realising that one! :smile: Will fix it :+1: